client/my-sites/stats/site.jsx

```diff
@@ -368,20 +374,36 @@ class StatsSite extends Component {
 					</div>
 				</div>
 				{ /** Promo Card is disabled for Odyssey because it doesn't make much sense in the context, which also removes an API call to `plugins`. */ }
-				{ ! isOdysseyStats && (
-					<div className="stats-content-promo">
-						<PromoCardBlock
-							productSlug="wordpress-seo-premium"
-							impressionEvent="calypso_stats_wordpress_seo_premium_banner_view"
-							clickEvent="calypso_stats_wordpress_seo_premium_banner_click"
-							headerText={ translate( 'Increase site visitors with Yoast SEO Premium' ) }
-							contentText={ translate(
-								'Purchase Yoast SEO Premium to ensure that more people find your incredible content.'
-							) }
-							ctaText={ translate( 'Learn more' ) }
-							image={ wordpressSeoIllustration }
-							href={ `/plugins/wordpress-seo-premium/${ slug }` }
-						/>
+				{ ! isOdysseyStats && isJetpackNonAtomic && (
+					<div className="stats__promo-container">
+						<div className="stats__promo-card">
+							<MobilePromoCard className="stats__promo-card-apps" />
+						</div>
+					</div>
+				) }
+				{ ! isJetpackNonAtomic && (
+					<div className="stats__promo-container">
+						<div className="stats__promo-card">
+							<DotPager className="stats__promo-pager">
+								<div>
+									<PromoCardBlock
+										productSlug="wordpress-seo-premium"
+										impressionEvent="calypso_stats_wordpress_seo_premium_banner_view"
+										clickEvent="calypso_stats_wordpress_seo_premium_banner_click"
+										headerText={ translate( 'Increase site visitors with Yoast SEO Premium' ) }
+										contentText={ translate(
+											'Purchase Yoast SEO Premium to ensure that more people find your incredible content.'
+										) }
+										ctaText={ translate( 'Learn more' ) }
+										image={ wordpressSeoIllustration }
+										href={ `/plugins/wordpress-seo-premium/${ slug }` }
+									/>
+								</div>
+								<div>
+									<MobilePromoCard className="stats__promo-card-apps" />
+								</div>
```

Comment:
```suggestion
								<PromoCardBlock
									productSlug="wordpress-seo-premium"
									impressionEvent="calypso_stats_wordpress_seo_premium_banner_view"
									clickEvent="calypso_stats_wordpress_seo_premium_banner_click"
									headerText={ translate( 'Increase site visitors with Yoast SEO Premium' ) }
									contentText={ translate(
										'Purchase Yoast SEO Premium to ensure that more people find your incredible content.'
									) }
									ctaText={ translate( 'Learn more' ) }
									image={ wordpressSeoIllustration }
									href={ `/plugins/wordpress-seo-premium/${ slug }` }
								/>
								<MobilePromoCard className="stats__promo-card-apps" />
```

No need for wrapping `div`s.